x-pack/plugins/upgrade_assistant/tests_client_integration/indices.test.ts

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
Most of the test coverage in UA is done by jest snapshots of individual components. This file only tests the new "fix index settings" flow. As we continue to make changes to UA, we'll slowly add more CITs and move away from the jest snapshots.